ngraph/core/src/function.cpp

```diff
@@ -101,7 +101,8 @@ Function::Function(const OutputVector& results,
 
 void Function::validate_nodes_and_infer_types()
 {
-    OV_ITT_SCOPED_TASK(itt::domains::nGraph, "Function::validate_nodes_and_infer_types");
+    OV_ITT_SCOPED_TASK(ngraph::itt::domains::nGraphPass_LT,
```

Comment:
Why we use load transformation domain here?

Can we use nGraph domain?

Reply:
This LT (load time) counter is required to measure the time of the first inference.

Reply:
It is not correct because we can call validate_nodes_and_infer_types not only in the moment of first inference.

Why cannot we use nGraph domain?